packages/bpk-stylesheets/index.js

```diff
@@ -17,6 +17,7 @@
  */
 
 import './index.scss';
+import '../bpk-component-badge/src/BpkBadge.module.scss';
```

Comment:
Here a human gets to order the component scss, this will define the order in the produced stylesheet.